addons/l10n_fi/data/res_country_group_data.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- Since account.fiscal.position.template doesn't have country_id -->
+    <!-- but account.fiscal.position has, this has to be manually set for -->
+    <!-- the intra-community fiscal position. -->
+    <record id="eu_wo_finland" model="res.country.group">
```

Comment:
It's so that you can set it on the fiscal position called "Intra-community", which roughly means other European Union countries without Finland. But you are correct, it's not very useful when it's undocumented.

I think we have at least three options here:
1. I create a PR that adds `country_group_id` and `country_id` to the `account.fiscal.position.template` model and passes those values to `account.fiscal.position` upon instantiation (I mean, why not?). Then I add this country group to the appropriate fiscal position template in the `country_group_id` field.
1. I create instructions to the description HTML file what to do right after installing this module. So, go to fiscal positions, check "Detect Automatically" and select this country group in the "Country Group" field.
1. I wipe this record away from the PR and we let the users do everything by hand.